gemini/function-calling/function_calling_data_structures.ipynb

```diff
@@ -0,0 +1,745 @@
+{
```

Comment:
<p>Maybe <code>Function Calling in Vertex AI</code> for the intro? Not sure what the branding is  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
Thanks! I added more detail to the intro and changed this to `Function Calling in Gemini`.